apps/content_filters/filter_condition/filter_condition_resource.py

```diff
@@ -75,3 +76,7 @@ class FilterConditionResource(Resource):
     privileges = {'POST': 'content_filters',
                   'PATCH': 'content_filters',
                   'DELETE': 'content_filters'}
+
+    def on_init(self):
+        for vocabulary in get_resource_service('vocabularies').get(req=None, lookup={}):
+            self.schema['field']['allowed'].append(vocabulary[config.ID_FIELD])
```

Comment:
not sure we should do this on init, it would need a restart when there is a new field added later

Reply:
Yep, wanted to add this but I forgot. Please check the latest update, now the filter allowed values are updated on vocabulary create/delete.

Reply:
I think the syncing won't work because it will only sync it in the process which did the changes, but not for other processes. would be probably better to check the field in the service